apps/condo/domains/ticket/schema/TicketStatus.js

```diff
@@ -7,8 +7,8 @@ const { GQLListSchema } = require('@core/keystone/schema')
 const { historical, versioned, uuided, tracked, softDeleted } = require('@core/keystone/plugins')
 const { SENDER_FIELD, DV_FIELD } = require('@condo/domains/common/schema/fields')
 const access = require('@condo/domains/ticket/access/TicketStatus')
+const { COMMON_AND_ORGANIZATION_OWNED_FIELD } = require('../../../schema/_common')
 const { TICKET_STATUS_TYPES } = require('../constants')
-const { COMMON_AND_ORGANIZATION_OWNED_FIELD } = require('@condo/domains/organization/schema/fields')
```

Comment:
use absolute imports pleas

Reply:
It's out of scope of this pull request.
I will fix it in another pull requests in further refactoring.